lib/ansible/modules/crypto/openssh_keypair.py

```diff
@@ -240,6 +240,7 @@ def dump(self):
             # On removal this has no value
             'fingerprint': self.fingerprint[1] if self.fingerprint else '',
             'public_key': self.public_key,
+            'comment': self.comment if self.comment else '',
```

Comment:
Probably also needs documentation in the `RETURN` description around line 90.

Reply:
Done